.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+**Steps to reproduce and a minimal demo of the problem**
```

Comment:
I'm not sure what's going on here... I pushed the latest version of develop (which is sync'd to master), but it's still showing all your previous changes in the PR...